src/operator/combineLatest.ts

```diff
@@ -13,10 +13,10 @@ import {isArray} from '../util/isArray';
  * @returns {Observable} an observable of other projected values from the most recent values from each observable, or an array of each of
  * the most recent values from each observable.
  */
-export function combineLatest<R>(...observables: Array<Observable<any> |
+export function combineLatest<T, R>(...observables: Array<any | Observable<any> |
```

Comment:
does combinelatest allowed to accept other than `Observable` and `Array<Observable>`?